mithril-aggregator/Dockerfile

```diff
@@ -37,7 +29,7 @@ RUN /app/target/release/mithril-aggregator --version
 ###############################
 # STEP 2: build a small image
 ###############################
-FROM ubuntu:22.04
+FROM debian:11-slim
```

Comment:
why isn't the `ubuntu-20.04` image used (the same as the CI)?

Reply:
Two reasons:

- I want to use debian in the dockerfiles as "compatibility test" to check that our ubuntu 20.04 built binaries are compatible with a later version of debian.
- this allow for the "dev" dockerfiles (the one without the `.ci` suffix) to use the [rust builder images](https://hub.docker.com/_/rust/) that are debian based, allowing a simplification of the builder step.